MODULE.bazel

```
@@ -1,8 +1,24 @@
 module(
-    name = "buildbuddy_toolchain",
-    version = "0.0.0",
+    name = "toolchains_buildbuddy",
+    version = "0.0.1",
     repo_name = "io_buildbuddy_buildbuddy_toolchain",
 )
 
 bazel_dep(name = "rules_cc", version = "0.0.9")
 bazel_dep(name = "platforms", version = "0.0.10")
+
+buildbuddy = use_extension("//:extensions.bzl", "buildbuddy")
+use_repo(buildbuddy, "buildbuddy_toolchain")
+
+register_toolchains(
+    "//toolchains/cc:ubuntu_gcc",
+    "//toolchains/cc:windows_msvc",
+)
+
+register_execution_platforms(
+    "//platforms:linux_x86_64",
+    "//platforms:linux_arm64",
+    "//platforms:darwin_x86_64",
+    "//platforms:darwin_arm64",
+    "//platforms:windows_x86_64",
```

```suggestion
    "//platforms:all",
```
More maintainable in the long run.